stdlib/sql/to_test.go

```diff
@@ -309,82 +308,60 @@ func TestToSQL_Process(t *testing.T) {
 }
 
 func TestToSql_NewTransformation(t *testing.T) {
-	testCases := []struct {
```

Comment:
Same, test case did not change, just using the new harness.
https://github.com/influxdata/flux/blob/f3e476b1ff1941ee636d53ec4506451a63b9c3ec/stdlib/sql/to_test.go#L310